WINDOWS.md

```diff
@@ -0,0 +1,58 @@
+## Visual Studio and Ninja Multi-Config Build Instructions
+
+The build use CMake for build system generation and Conan 2 for dependency management.  Prerequisites for building are:
+ - A supported C++ compiler
+   * GCC, 11.x and later
+   * Clang, 14.x and later
+   * MSCV, 2022 and later
+ - Python
+ - Git
+
+Visual Studio 2022 can be downloaded from Microsoft [website](https://visualstudio.microsoft.com/vs/community/)
+A minimum working installation to support the project build will have the following details:
+![](./VisualStudioInstall.png)
```

Comment:
This image should be moved to a specific folder for images rather than be left in the root (we want to keep the roots as uncluttered as possible).  Let's move this to `<root>/docs/images`, and keep all other images there for now.